content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -61,6 +61,34 @@ This section describes the `PrismaClient` constructor and its parameters.
 
 - Parameters are validated at runtime.
 
+### <inlinecode>adapter</inlinecode>
+
+Defines an instance of a [driver adapter](/concepts/components/drivers#driver-adapters). See also [Drivers](/concepts/components/drivers) <span class="concept"></span>.
+
+This is available from version 5.4.0 and newer.
+
+This feature is currently in Preview behind the `driverAdapters` feature flag.
+
+#### Example
+
+The example below uses the [Neon driver adapter](/guides/database/neon#how-to-use-the-neon-serverless-driver-with-prisma-preview)
+
+```ts
+import { Pool, neonConfig } from '@neondatabase/serverless'
+import { PrismaNeon } from '@prisma/adapter-neon'
+import { PrismaClient } from '@prisma/client'
+import dotenv from 'dotenv'
+import ws from 'ws'
+
+dotenv.config()
+neonConfig.webSocketConstructor = ws
+const connectionString = `${process.env.DATABASE_URL}`
+
+const pool = new Pool({ connectionString })
+const adapter = new PrismaNeon(pool)
+const prisma = new PrismaClient({ adapter })
+```
+
```

Comment:
Had I not suggested to move it to the end of the options list?
(Sorry if I really wrote "move to top".)
Logically it makes most sense to me at the end of the list for now.

Reply:
Why should it be at the bottom instead of the top of the list? 🤔